comfyui_to_python.py

```diff
@@ -431,18 +447,21 @@ def execute(self):
             None
         """
         # Step 1: Import all custom nodes
-        import_custom_nodes()
+        init_builtin_extra_nodes()
```

Comment:
I tried using `init_extra_nodes` instead of `init_builtin_extra_nodes`, do you know what the difference is?